docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -23,6 +23,27 @@
 |spring.sleuth.integration.websockets.enabled | `true` | Enable tracing for WebSockets.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.count-query | `false` | Creates listener to count queries. @see ProxyDataSourceBuilder#countQuery() @see QueryCountHolder
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.json-format | `false` | Use json output for logging query. @see ProxyDataSourceBuilder#asJson()
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.logging |  | Logging to use for logging queries.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.multiline | `true` | Use multiline output for logging query. @see ProxyDataSourceBuilder#multiline()
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.query.enable-logging | `true` | Enable logging all queries to the log.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.query.log-level | `DEBUG` | Severity of query logger.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.query.logger-name |  | Name of query logger.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.slow-query.log-level | `WARN` | Severity of slow query logger.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.slow-query.logger-name |  | Name of slow query logger.
+|spring.sleuth.jdbc.decorator.datasource.datasource-proxy.slow-query.threshold | `300` | Number of seconds to consider query as slow.
+|spring.sleuth.jdbc.decorator.datasource.enabled | `true` | Enables data source decorating.
+|spring.sleuth.jdbc.decorator.datasource.excluded-beans |  | Beans that won't be decorated.
+|spring.sleuth.jdbc.decorator.datasource.includes |  | Which types of tracing we would like to include.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.custom-appender-class |  | Class file to use (only with logging=custom). The class must implement {@link com.p6spy.engine.spy.appender.FormattedLogger}.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.enable-logging | `true` | Enables logging JDBC events.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.log-file | `spy.log` | Name of log file to use (only with logging=file).
+|spring.sleuth.jdbc.decorator.datasource.p6spy.log-filter.pattern |  | Use regex pattern to filter log messages. Only matched messages will be logged.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.log-format |  | Custom log format.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.logging |  | Logging to use for logging queries.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.multiline | `true` | Enables multiline output.
+|spring.sleuth.jdbc.decorator.datasource.p6spy.tracing.include-parameter-values | `true` | Report the effective sql string (with '?' replaced with real values) to tracing systems. <p> NOTE this setting does not affect the logging message.
```

Comment:
It doesn't make much sense to have all these properties in Sleuth, maybe some defaults need to be changed to `false`, like `spring.sleuth.jdbc.decorator.datasource.p6spy.enable-logging`/`spring.sleuth.jdbc.decorator.datasource.datasource-proxy.query.enable-logging`.  These defaults made sense when it was separate library with Sleuth as only one of the functions, but when someone enables tracing, and if I remember correctly traces are created with some probability, then starting to see logs for each and every query/commit/rollback will be a bit strange for Sleuth users.

Also maybe `spring.sleuth.jdbc.decorator.datasource.*` can be replaced with just `spring.sleuth.jdbc.*`, WDYT?

Reply:
> I remember correctly traces are created with some probability, then starting to see logs for each and every query/commit/rollback will be a bit strange for Sleuth users.

Traces are always created. With certain probability they are sent to a reporting system.

> Also maybe spring.sleuth.jdbc.decorator.datasource.* can be replaced with just spring.sleuth.jdbc.*, WDYT?

great idea

Reply:
> maybe some defaults need to be changed to false, like spring.sleuth.jdbc.decorator.datasource.p6spy.enable-logging/spring.sleuth.jdbc.decorator.datasource.datasource-proxy.query.enable-logging.

Actually having those specific logging is pretty cool. Do you think that it should be an opt-in feature?

Reply:
it depends :) On my current project if query logging was suddenly enabled our logging cluster would just die. It would easily add 60% of logging traffic, unlike tracing which is throttled to some extent.

Reply:
Makes sense. I'll make it opt in.